deploy/src/main/deploy/azure/arm/honoInfrastructureDeployment.json

```diff
@@ -318,51 +330,51 @@
         },
         "mqttPublicIPAddress": {
             "type": "string",
-            "value": "[reference('honoKubernetesDeployment').outputs.mqttIpAddress.value]"
+            "value": "[reference('honoKubernetesDeployment', '2019-06-01').outputs.mqttIpAddress.value]"
```

Comment:
are you sure that the date needs to be in here?

Reply:
It is not a date, but API version. If missing the resource manager will throw an error like 

`"Azure Error: InvalidTemplate
Message: Deployment template validation failed: 'The template output reference to 'Microsoft.Resources/deployments/honoServiceBusDeployment' requires an API version. Please see https://aka.ms/arm-template for usage details.'."
`

Reply:
I see :-)